### ISSUE TITLE
CoolProp 6.6.0 not working on ARM architecture

### ISSUE THREAD

**Issue by mechEman** [CoolProp/CoolProp#2414]
### Description

I have a MacBook Pro with the M2 Pro chip. I am running Parallels on it to have Windows. When I download the CoolProp 6.6.0 I cannot open the file because it said it does not work on x64 architecture. I have reached out to Parallels support and they were able to replicate the issue with an apple chip. They were, however, able to get it to work with parallels running on an intel chip.

### Steps to Reproduce

1. Download CoolProps 6.6.0 in windows on an x64 architecture. Preferably on Parallels with an apple chip.
2. View the error message that pops up when trying to access the download.

**Expected behavior:** I expected the download to work like normal on windows.

**Actual behavior:** error message for x64 architecture. 

### Versions

**CoolProp Version:** 6.6.0  
**Operating System and Version:** windows 11 pro running on Parallels. MacBook Pro M2 Pro chip.
**Access Method:** Downloaded from the internet

**Comment by ibell** [comment]
What file did you download,  and what architecture is operating inside your
parallels? By the way,  why not just use your fast nice M2 Mac?  I do for
everything.

**Comment by mechEman** [comment]
Did you find a way to use CoolProps on excel for Mac? 

This is the exact link where I got it from: https://sourceforge.net/projects/coolprop/files/CoolProp/6.6.0/Installers/Windows/. 

My Parallels is operating on X64 ARM architecture

**Comment by ibell** [comment]
arm64 is not the same as x64, and we do not provide binaries for arm for
windows. You could build it yourself within your windows image following
the docs.

There are docs on hooking Excel and coolprop on macs,  but it is an
annoying process. I would instead recommend you use python,  much greater
power, and avoids a lot of the problems with Excel

**Comment by ganyu123456** [comment]
https://github.com/CoolProp/CoolProp/issues/2414#issuecomment-2361469087
I encountered the same problem. Since the ARM64 so file is not provided, can I hope the author can compile the ARM64 file?

**Comment by ibell** [comment]
Nope, but _you_ can, with free and/or open-source tools, following the instructions from here: http://coolprop.org/coolprop/wrappers/SharedLibrary/index.html#shared-library

**Comment by ibell** [comment]
I got the arm64 builds working. The binaries will be included with the next release, but for now, try this one: https://github.com/CoolProp/CoolProp/actions/runs/12216723140/artifacts/2289341143 (you'll want the files in Windows/64bit__arm64)

**Comment by ibell** [comment]
Fixed in version 6.7.0
